internal/cmd/beta/security-group/delete/delete.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
 	"github.com/stackitcloud/stackit-cli/internal/pkg/projectname"
 	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/client"
+	iaas_utils "github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/utils"
```

Comment:
It should be `iaasUtils` to follow Go's naming standard

Reply:
renamed